forge-gui/res/cardsfolder/s/singing_towers_of_darillium.txt

```diff
@@ -0,0 +1,8 @@
+Name:Singing Towers of Darillium
+ManaCost:no cost
+Types:Plane Darillium
+S:Mode$ Continuous | Affected$ Card.nonLand+YouOwn+withoutForetell | EffectZone$ Command | AffectedZone$ Hand | AddKeyword$ Foretell | Description$ Each nonland card in your hand without foretell has foretell. Its foretell cost is equal to its mana cost reduced by {2}. (During your turn, you may pay {2} and exile it from your hand face down. Cast it on a later turn for its foretell cost.)
```

Comment:
this doesn't seem to handle `Its foretell cost is equal to its mana cost reduced by {2}` ?